mkdocs.yml

```diff
@@ -4,11 +4,10 @@ site_url: https://handbook.civicactions.com/en/latest/
 theme:
   name: readthedocs
   highlightjs: true
+  collapse_navigation: true
```

Comment:
Discussed with @grugnog that we want to keep other changes in this PR but revert the `collapse_navigation` change.